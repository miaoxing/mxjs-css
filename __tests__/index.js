import {css} from '..';
import {css as originalCss} from '@fower/core';

describe('css', () => {
  test('css', () => {
    expect(originalCss).toBe(css);
  });

  test('preset', () => {
    expect(css('mb4')).toMatchSnapshot();
  });
});
