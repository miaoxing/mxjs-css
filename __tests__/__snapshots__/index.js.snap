// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`css preset 1`] = `"mb-16"`;
